local.yml

```diff
@@ -61,9 +61,13 @@ services:
     environment:
       - FLASK_DEBUG=1
       - FLASK_APP=src/factory.py
+      - RUN_SLOW_TESTS=0
```

Comment:
No need for this; surely it's implicit?

Adding it here actually implies it's *required* otherwise we run the slow tests.

Reply:
I thought it would make sense to add it here so people directly know where to change that "switch".